cpp/velox/utils/ConfigExtractor.cc

```diff
@@ -59,6 +59,17 @@ std::shared_ptr<facebook::velox::config::ConfigBase> getHiveConfig(
   const std::string kVeloxAwsSdkLogLevel = "spark.gluten.velox.awsSdkLogLevel";
   const std::string kVeloxAwsSdkLogLevelDefault = "FATAL";
 
+  // Whether to use proxy from env for s3 c++ client
```

Comment:
All the config in native backend should have an entry in GlutenConfig.scala, and these configs should add to function `getNativeBackendConf` like other S3 config. Otherwise, the config cannot send to native backend.

Reply:
Thanks a lot. Add these configurations into GlutenConfig.scala